dbpedia/dbPedia_Composers.js

```diff
@@ -152,41 +164,34 @@ function getResults(queryURL, callback) {
                 array = null;
```

Comment:
i think this null check also needs to be deleted. Because "array" maps to work later in the object, so this should be [] instead of null